src/scrapers/hapoalim.js

```diff
@@ -55,11 +55,15 @@ function convertTransactions(txns) {
       description: txn.activityDescription,
       status: txn.serialNumber === 0 ? TRANSACTION_STATUS.PENDING : TRANSACTION_STATUS.COMPLETED,
       memo,
+      activityTypeCode: txn.activityTypeCode,
```

Comment:
added those fields that I need from the website.
remember I saw somewhere you want all field of the scrapers to be shared?

Reply:
those are codes that the bank gives and it helps to identify later specific payments and also identify their category